packages/lit-html/src/directive.ts

```diff
@@ -110,10 +110,14 @@ export abstract class Directive {
   _$disconnectableChildren?: Set<Disconnectable>;
   // This property needs to remain unminified.
   //@internal
-  ['_$setDirectiveConnected']?(isConnected: boolean): void;
+  ['_$notifyDirectiveConnectionChanged']?(isConnected: boolean): void;
 
   constructor(_partInfo: PartInfo) {}
 
+  get _$isConnected() {
```

Comment:
Add a comment here about how this works and why it's here.

Reply:
Added a comment